README.md

```diff
@@ -139,7 +135,7 @@ Installation steps:
 10. (optional) Index your search database in solr server using  `bundle exec rake sunspot:reindex`
 11. Start rails with `passenger start` from the Rails root and open http://localhost:3000 in a web browser.
 12. Wheeeee! You're up and running! Log in with test usernames "user", "moderator", or "admin", and password "password".
-13. Run `rake test` to confirm that your install is working properly. For some setups, you may see warnings even if test pass; [see this issue](https://github.com/publiclab/plots2/issues/440) we're working to resolve.
```

Comment:
Oh yes those paperclip gem deprecation warnings. Ok then we should keep this. I will mention about this in that issue.